product-service/handlers/getProductById.js

```diff
@@ -8,21 +8,22 @@ const cors = {
 const buildResponse = (statusCode, body, headers) => ({statusCode, body: JSON.stringify(body), headers});
 
 const handler = async (event) => {
+  console.log('Incoming event: ', event);
   const id = event.pathParameters?.id;
 
   try {
-    const products = await getMockedProducts();
+    const { rows: products } = await query(`SELECT p.id, p.title, p.description, p.price, p.image, s.count FROM product p INNER JOIN stock s ON p.id = s.product_id WHERE p.id::text = $1`, [id]);
 
```

Comment:
`'SELECT product.*, stock.count FROM product LEFT JOIN stock ON stock.product_id=product.id WHERE id = $1', [id]`

Reply:
May be. I'll leave it as it is for now.